common/changes/@rushstack/real-node-module-path/fast-realpath_2024-11-20-03-42.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/real-node-module-path",
```

Comment:
It seems like it would make sense to just stick this in node-core-library.

Reply:
Yeah I've come around to this. I originally thought it would make more sense to have as a separate package, but given the size I can totally see this being a separate method in `FileSystem` or even just triggered on an arg to `FileSystem.getRealPath*` methods